### PR TITLE
Fixup iron165r2

### PR DESCRIPTION
**Pull request by waffle87**
## Description
Fixes some minor issues.
@Gondolindrim with the current file structure, this needs a `DEFAULT_FOLDER`. For the time being, I have set it to `f411`.

## Types of Changes

- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by waffle87** [comment]
> Whats the requirement for moving `RGBLED_NUM` and `LED_CAPS_LOCK_PIN`?

First thought was to move pin defines to MCU rules.mk level, before thinking of `DEFAULT_FOLDER`. Reverted.

**Comment by drashna** [comment]
best way is probably to remove the rules.mk from the main folder, and add it's values to the f072 and f411 folder's rules.mk.

File: keyboards/smithrune/iron165r2/config.h [hunk 1]
```diff
@@ -45,6 +45,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //#define LINE_RGBS
 //#define RUNE_RGBS
 #define LUKE_RGBS
+#define RGBLED_NUM 22
```
Comment (zvecr) [inline keyboards/smithrune/iron165r2/config.h:48]:
```suggestion
```

File: keyboards/smithrune/iron165r2/f072/config.h [hunk 2]
```diff
@@ -27,7 +27,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define BACKLIGHT_ON_STATE 1
 
 #define RGB_DI_PIN B15
```
Comment (zvecr) [inline keyboards/smithrune/iron165r2/f072/config.h:29]:
```suggestion
#define RGB_DI_PIN B15
#define RGBLED_NUM 22
```

File: keyboards/smithrune/iron165r2/f411/config.h [hunk 3]
```diff
@@ -28,7 +28,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define STM32_HSECLK 8000000
 
 #define RGB_DI_PIN B15
```
Comment (zvecr) [inline keyboards/smithrune/iron165r2/f411/config.h:30]:
```suggestion
#define RGB_DI_PIN B15
#define RGBLED_NUM 22
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/smithrune/iron165r2/f072/rules.mk [hunk 4]
```diff
@@ -1,4 +1,24 @@
...
+NKRO_ENABLE = yes           # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = no
+ENCODER_ENABLE = no
 BACKLIGHT_ENABLE = yes
 BACKLIGHT_DRIVER = pwm
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (zvecr) [inline keyboards/smithrune/iron165r2/f072/rules.mk:24]: Any reason this wasnt added to the f411 rules too?

Comment (waffle87) [reply keyboards/smithrune/iron165r2/f072/rules.mk:24]: Sorry, didn't think it was necessary for F411. Resolved.

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**